site/_data/chrome.js

```diff
@@ -89,28 +90,52 @@ module.exports = async function buildVersionInformation() {
     }
 
     for (const {channel, version} of versions) {
+      if (!['stable', 'beta'].includes(channel)) {
+        continue;
+      }
       const majorVersion = extractMajorVersion(version);
       if (majorVersion === 0) {
         continue;
       }
-      channelRelease[channel] = Math.max(
-        channelRelease[channel] ?? 0,
-        majorVersion
-      );
+      raw[channel] = Math.max(raw[channel] ?? 0, majorVersion);
     }
   }
 
-  /** @type {{[channel: string]: ChromeReleaseData}} */
-  const channels = {};
+  // Sanity-check that we have 'stable', and if we have 'beta', that it's one or two higher.
+  // Create the virtual 'pending' if it's two higher.
+  if (!('stable' in raw) || raw['beta'] <= raw['stable']) {
+    console.warn(raw);
+    throw new Error('bad Chrome release data');
+  }
+  if (!('beta' in raw)) {
+    raw['beta'] = raw['stable'] + 1;
+  }
+  const {beta, stable} = raw;
+  if (beta === stable + 1) {
+    // great
+  } else if (beta === stable + 2) {
+    // This can occur if Chrome does something weird with releases, basically the previous beta
+    // is about to be released but it's not done yet.
+    raw['pending'] = stable + 1;
+  } else {
+    console.warn(raw);
+    throw new Error('bad Chrome release data, beta is >+2 from stable');
```

Comment:
One note here, I want a lighter-touch type of an error: crash in prod but warn/defaults in dev. Perhaps that's for another PR. 🤷